### DISCUSSION TITLE
iHelp (15/06/2024)

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#53] [Esami]
In questa discussion a tema il progetto "iHelp", oggetto del testo di esame del 15 Giugno 2024, discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**Comment by daromacarlo** [comment]
Mia proposta di soluzione, fatemi sapere se ci sono errori (orrori)
[15 giugno.pdf](https://github.com/user-attachments/files/16068892/15.giugno.pdf)
